tinydb/storages.py

```diff
@@ -134,3 +134,44 @@ def read(self):
 
     def write(self, data):
         self.memory = data
+
+
+class EphemeralJSONStorage(Storage):
```

Comment:
IMO you should subclass JSONStorage instead, that way you just need to override `write` and on a `read` you can do something like:

```
def read(self):
    if self.memory is not None:
        return self.memory
    return super().read(self)
```

Reply:
Thanks! I should have done so.